src/main.rs

```diff
@@ -31,6 +31,9 @@ struct EvgExpansions {
     pub task_name: String,
     /// ID of Evergreen version running.
     pub version_id: String,
+    /// True if the patch is a patch build.
+    #[serde(default)]
```

Comment:
I added the `serde(default)` bit in case the `is_patch` variable isn't provided. I ran the task-generator locally with and without the `is_patch` variable in the expansions file and it seemed to accordingly include / omit the `--skipTestsCoveredByMoreComplexSuite` flag when calling Resmoke.